### PR TITLE
Added eslint in the project

### PR DESCRIPTION
**Pull request by Dhroov7**

### CODE REVIEW

File: scripts/test.sh [hunk 1]
```diff
@@ -1,14 +1,32 @@
 #!/bin/bash
 # ----------------------------------------------------------------------------------------------------------------------
-# This script is intended to execute unit tests.
+# This script is intended to be executed as all-inclusive test and artefact creation (excluding packaging and release.)
```
Comment (abhijitkane) [inline scripts/test.sh]: @Dhroov7 None of this is relevant for this project

File: package.json [hunk 2]
```diff
@@ -15,13 +15,22 @@
     "uuid": "3.2.1"
   },
   "devDependencies": {
+    "editorconfig": "^0.15.3",
+    "eslint": "5.16.0",
+    "eslint-plugin-jsdoc": "3.8.0",
+    "eslint-plugin-mocha": "5.2.0",
+    "eslint-plugin-security": "1.4.0",
     "expect.js": "0.3.1",
+    "istanbul": "^0.4.5",
```
Comment (abhijitkane) [inline package.json]: @Dhroov7 Why do we need istanbul and nyc?

File: scripts/test.sh [hunk 3]
```diff
@@ -1,14 +1,26 @@
 #!/bin/bash
 # ----------------------------------------------------------------------------------------------------------------------
-# This script is intended to execute unit tests.
+# This script executes all non time-bound tests, excluding browser tests and performance tests.
```
Comment (abhijitkane) [inline scripts/test.sh]: browser tests? Why do we need to specify things we're not testing?

File: package.json (hunk 2 above)
Comment (Dhroov7) [reply package.json]: We can remove istanbul but it gives code coverage report.
I've removed nyc.

**Pull request merged**